src/java.desktop/windows/classes/sun/awt/windows/WEmbeddedFrame.java

```diff
@@ -189,13 +186,7 @@ protected static int getPrintScaleFactor() {
         if (printScale == null) {
             // if no system property is specified,
             // check for environment setting
-            printScale = AccessController.doPrivileged(
-                new PrivilegedAction<String>() {
-                    public String run() {
-                        return System.getenv("JAVA2D_PLUGIN_PRINT_SCALE");
-                    }
-                }
-            );
+            printScale = System.getenv("JAVA2D_PLUGIN_PRINT_SCALE");
```

Comment:
Is the `@SuppressWarnings("removal")` on this method still needed?

Reply:
removed